src/libraries/System.Runtime.Intrinsics.Experimental/ref/System.Runtime.Intrinsics.Experimental.cs

```diff
@@ -210,6 +210,18 @@ public abstract partial class AdvSimd : System.Runtime.Intrinsics.Arm.ArmBase
         public static System.Runtime.Intrinsics.Vector64<uint> CompareTest(System.Runtime.Intrinsics.Vector64<uint> left, System.Runtime.Intrinsics.Vector64<uint> right) { throw null; }
         public static System.Runtime.Intrinsics.Vector64<double> DivideScalar(System.Runtime.Intrinsics.Vector64<double> left, System.Runtime.Intrinsics.Vector64<double> right) { throw null; }
         public static System.Runtime.Intrinsics.Vector64<float> DivideScalar(System.Runtime.Intrinsics.Vector64<float> left, System.Runtime.Intrinsics.Vector64<float> right) { throw null; }
+        public static System.Runtime.Intrinsics.Vector128<float> FusedMultiplyAdd(System.Runtime.Intrinsics.Vector128<float> acc, System.Runtime.Intrinsics.Vector128<float> left, System.Runtime.Intrinsics.Vector128<float> right) { throw null; }
```

Comment:
We may want to discuss the ordering in API review.

The scalar versions are `FMADD <Sd>, <Sn>, <Sm>, <Sa>`, where it does `Sd = Sa + Sn * Sm` (registers ordered `left, right, acc`)

The vector versions are `FMLA <Vd>, <Vn>, <Vm>` where it does `Vd = Vd +  Vn * Vm` (registers ordered `acc, left, right` as you have here)

x86 intrinsics and the public `Math.FusedMultiplyAdd` methods are all `left, right, acc`